blocks/image-compare/src/edit.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * WordPress dependencies
+ */
+import { InspectorControls, RichText } from '@wordpress/block-editor';
+import { PanelBody, RadioControl, Placeholder } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import UploadPlaceholder from './upload-placeholder';
+
+/* global juxtapose */
+
+const edit = ( { attributes, isSelected, setAttributes } ) => {
+	const { imageBefore, imageAfter, caption, orientation } = attributes;
+
+	const RadioControlWithState = ( props ) => {
```

Comment:
I'm pretty sure I messed up this bit, with regards to setting the orientation of the interface. If you can help me get this working, I'd appreciate it.

Reply:
Fixed.

Reply:
You rock. Let the record show I owe you a beverage.